rednotebook/external/text2rednotebook.py

```diff
@@ -0,0 +1,145 @@
+"""
+Mass import diary/journal entries from markdown/plain text/odt into rednotebook
+"""
+
+import argparse
+import sys
+import re
+import datetime
+from odf import text
+from odf.opendocument import load
+sys.path.append('/usr/share/rednotebook')
+
+# must import rednotebook after munging path in order to find it, as it is
+# not (at least in Debian) installed in the default python path
+import rednotebook.storage  # pylint: disable=wrong-import-position, unused-import
+
+def main():
+    """ parse commandline arguments & process text """
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-n', '--dry-run', action='store_true')
+    parser.add_argument('--echo-entries', action='store_true')
+    parser.add_argument('--existing', choices=['overwrite', 'append', 'skip', 'error'],
+                        default='error')
+    parser.add_argument('--no-list-missing-entries', action='store_true')
+    parser.add_argument('infile')
```

Comment:
Can you add some information to the options with the `help` argument?

Reply:
Done